source/blog/2016-05-30-dropping-dropbox.html.markdown

```diff
@@ -0,0 +1,97 @@
+
+## Overview
+
+Okay, so BitTorrent Sync is a service that runs on computers and synchronizes files across them using BitTorrent. Cool. Their site isn't clear on this, but you [need to buy a license](https://getsync.com/features#compare) to sync across multiple devices. It's $40/year, but has a 30-day trial (that they only tell you about after you download the app). Yeah, documentation needs some work.
+
+BitTorrent Sync is a really impressive service that provides a _tonne_ of awesome features. I'm probably not going to use most of them, but crucially, it provides secure syncing of files across computers and smartphones.
+
+The BitTorrent protocol is inherently decentralized, so naturally syncing with BitTorrent is decentralized too. If you have two computers online, and add a file to one, then it'll get transferred to the other over a secure, encrypted connection [?]. But consider the following scenario:
+
+1. Computer A is on, B is off.
+1. Add a file to A.
+1. Turn A off.
+1. Turn B on.
+1. The file isn't sync'd to B, because A isn't online to send it.
+
+The solution to this is to have a computer that's running BitTorrent Sync that's always on. Unless you have a home server or colocated machine, you're going to have to use a virtual server. That's where Amazon Web Services comes in.
+
+AWS is super-popular, but managing it is so infamously unfriendly that there's a [AWS in Plain English](https://www.expeditedssl.com/aws-in-plain-english) site to help. We're going to use EC2, their virtual server service. The best blog post I found to help this is [here](http://meltingice.net/2015/07/27/how-to-run-bittorrent-sync-on-aws/), and it's an excellent reference on how to set up BitTorrent Sync on a new EC2 instance (basically a virtual server). You get a year of free service from AWS, and after that it's about $10/month (when billed hourly – you can save money by [billing monthly or yearly](https://aws.amazon.com/ec2/pricing/)). 
+
+The post has some great suggestions, like using a random, pre-configured port for BitTorrent Sync, and allowing incoming connections only on that one port and port 80 for administration. However, there are a few aspects to the post I think can be improved upon:
+
+1. <p>The post recommends opening port 80 to the entire internet, exposing the BitTorrent Sync dashboard (and consequently, access to your files [?]) to anyone who provides the username and password, which are stored on the server in plaintext.</p><p>Further complicating this problem is the fact that to administer BitTorrent Sync, you send your username and password to the server over port 80 _unencrypted_, so anyone monitoring your traffic can see the auth details in plaintext [?]. Yikes.</p>
```

Comment:
Hmm, even if the config is saved elsewhere on the system by default, their [documentation shows](http://help.getsync.com/hc/en-us/articles/204762689-Running-Sync-in-configuration-mode#overview) the username and password for the web UI dashboard in a JSON file. Yeah, pretty not great. I'll have to do some thinking. And the post doesn't make it clear that I'm still evaluating the software and haven't moved fully over to it yet.